op-batcher/batcher/driver.go

```diff
@@ -181,10 +180,18 @@ func (l *BatchSubmitter) StartBatchSubmitting() error {
 		l.Log.Warn("Throttling loop is DISABLED due to 0 throttle-interval. This should not be disabled in prod.")
 	}
 
-	l.wg.Add(3)
-	go l.receiptsLoop(l.wg, receiptsCh)                                            // ranges over receiptsCh channel
-	go l.publishingLoop(l.killCtx, l.wg, receiptsCh, publishSignal)                // ranges over publishSignal, spawns routines which send on receiptsCh. Closes receiptsCh when done.
-	go l.blockLoadingLoop(l.shutdownCtx, l.wg, pendingBytesUpdated, publishSignal) // sends on pendingBytesUpdated (if throttling enabled), and publishSignal. Closes them both when done
+	if l.Config.UseEspresso {
+		l.wg.Add(4)
+		go l.receiptsLoop(l.wg, receiptsCh) // ranges over receiptsCh channel
+		go l.espressoBatchQueueingLoop(l.shutdownCtx, l.wg)
+		go l.espressoBatchLoadingLoop(l.shutdownCtx, l.wg, publishSignal)
```

Comment:
In Espresso mode we have `batchLoadingLoop` and then `publishingLoop` while in standard OP mode we have `publishingLoop` and then `blockLoadingLoop`. Is there a specific reason to switch the order of these calls or does the order not matter?

Reply:
The order is irrelevant